src/Containers/DenseAxisArray.jl

```diff
@@ -60,9 +60,15 @@ end
 
 function Base.getindex(
     x::_AxisLookup{Dict{K,Int}},
-    key::K,
-) where {K<:AbstractVector}
-    return x.data[key]
+    key::T,
+) where {T<:AbstractVector,K>:T}
```

Comment:
Can't we keep also the efficient method in case `T` is `K` ?

Reply:
I don't think so, because the case `T = K = Vector{Any}` is ambiguous. There's probably a bunch of similar types.